pkg/util/trace/impl/motrace/report_statement.go

```diff
@@ -36,6 +37,71 @@ var NilTxnID [16]byte
 var NilSesID [16]byte
 
 // StatementInfo implement export.IBuffer2SqlItem and export.CsvFields
+
+var _ IBuffer2SqlItem = (*StatementInfo)(nil)
+
+func StatementInfoNew(i Item, ctx context.Context) Item {
+	if s, ok := i.(*StatementInfo); ok {
+		// process the execplan
+		s.ExecPlan2Json(ctx)
+		// remove the plan
+		s.jsonByte = nil
+		s.ExecPlan = nil
+
+		// remove the TransacionID
+		s.TransactionID = NilTxnID
+		s.StatementTag = ""
+		s.StatementFingerprint = ""
+		s.Error = nil
+		s.RowsRead = 0
+		s.BytesScan = 0
+		s.ResultCount = 0
+		return s
+	}
+	return nil
+}
+func StatementInfoUpdate(existing, new Item) {
+
+	e := existing.(*StatementInfo)
+	n := new.(*StatementInfo)
+	// update the stats
+	e.Duration += n.Duration
+	e.Statement += n.Statement + "; "
```

Comment:
Don't do this, just update a counter.    Each statement is not interesting.

Reply:
@gavinyue Has this one been solved ? this one may will affect the aggr improvement.